### DISCUSSION TITLE
DDC working on Monitorian but not on MonitorControl / BetterDisplay

### DISCUSSION THREAD

**Discussion by parichay28** [MonitorControl/MonitorControl#1599] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

Was looking for a brightness control solution for windows & found [Monitorian](https://github.com/emoacht/Monitorian)

What caught my attention is that Monitorian was able to change the actual brightness on my monitor (LG 27QN600) instead of using a layer over screen.

I use this monitor with both macbook (m1 2020 model) & windows PC. So far, I have been under assumption that my monitor don't support DDC & that's why [MonitorControl](https://github.com/MonitorControl/MonitorControl) & [BetterDisplay](https://github.com/waydabber/BetterDisplay) reduce brightness by adding a layer on top of the screen to stimulate brightness reduction (which at times doesn't feel natural, depending on screen's actual brightness)

But with Monitorian, saw the actual brightness increasing/decreasing & was able to verify the same in my monitor OSD controls.

Skimmed through a [few issues with mention of Monitorian](https://github.com/MonitorControl/MonitorControl/issues?q=%22Monitorian%22) & read monitor troubleshooting. Found that GPU might have a role to play here (using Nvidia GeForce RTX 3060 on windows). Also can see `Automatically Adjust Brightness: No` in mac system report 😿 
<br />
<img width="899" alt="image" src="https://github.com/user-attachments/assets/b1fac809-230f-49f8-b745-ee8bfe160e68">
<br />
Would there be a way make this work on macos as well? Feel free to redirect to right issue / discussion if this has been asked before ❤️

**Comment by waydabber** [comment]
Hi there - how is the display connected? BetterDisplay for sure should control the brightness via DDC. Did you try detecting DDC capabilities with the app? If so, what was the result? Can you show me a screenshot of what you see under the Settings/Displays in BetterDisplay?

**Reply by parichay28** [reply]
Hey @waydabber so I'm on a custom scaled resolution (HiDPI). `Potentially DDC Compatible` does say `yes` on BetterDisplay. The connection is via an HDMI to HDMI cable (via [Portronics C-Konnect](https://www.amazon.in/Portronics-C-Konnect-Adapter-Project-Projector/dp/B083DY9RXW))

On turning off "Enable software-based video adjustments" or "Combined brightness". The dimming no longer happens indicating s/w based dimming instead of DDC

LMK what exact settings would be more helpful & if you'd prefer to move this to BetterDisplay repo instead

<img width="582" alt="image" src="https://github.com/user-attachments/assets/b3e9bd8b-984a-44d7-99ef-db7e399067ff">
<br />
<img width="558" alt="image" src="https://github.com/user-attachments/assets/4b4008d4-2763-48c2-8a66-4f5f01685372">
<br />
<img width="561" alt="image" src="https://github.com/user-attachments/assets/70fd51cd-0128-49ee-8aee-6ea74e39ef74">

**Reply by parichay28** [reply]
<img width="588" alt="image" src="https://github.com/user-attachments/assets/a048abc9-272e-43d8-97f9-194282f3b76f">

Updated to the latest version. Found "detecting ddc capabilities" option, but this is stuck here after a while

**Reply by parichay28** [reply]
Okay so this took a while, here's the latest info

<img width="583" alt="image" src="https://github.com/user-attachments/assets/d6ea0540-bc5e-49d2-ab99-0ee3c22ec528">

**Reply by waydabber** [reply]
Does control actually work? This only means that capabilities strings cannot be read, but many displays do not provide a proper string despite it's required by the standard - while DDC control still works.

**Reply by waydabber** [reply]
(but yes, you may of course open a discussion about this under BetterDisplay if you wish - I can't transfer discussions between repos).

**Reply by parichay28** [reply]
> Does control actually work?

Unfortunately no. Planning to get a type C to DP cable & ditch all adapters to see if that helps.

**Comment by parichay28** [comment]
@waydabber Moved the discussion [here](https://github.com/waydabber/BetterDisplay/discussions/3342) for BetterDisplay. Will keep this open for the time being in case if this catches MonitorControl's team attention.

**Reply by waydabber** [reply]
Nice!
